core/src/main/java/org/pentaho/di/connections/utils/EncryptUtils.java

```diff
@@ -2,7 +2,7 @@
  *
  * Pentaho Data Integration
  *
- * Copyright (C) 2019 by Hitachi Vantara : http://www.pentaho.com
+ * Copyright (C) 2019 - 2024 by Hitachi Vantara : http://www.pentaho.com
```

Comment:
No fault of your own, previous author didn't write unit tests for `EncryptUtils.java`

Can we add an jira story similar to https://hv-eng.atlassian.net/browse/BACKLOG-39259 to track this effort to add/increase unit tests?

Reply:
Not yet. Will create a story to track that.